sphinx/source/docs/user_guide/bokehjs.rst

```diff
@@ -280,6 +280,78 @@ and hover policy. Here is an example of a ``bar`` chart and the plot it generate
 
     plt.show(plt.gridplot([[p1, p2], [p3, p4]], {plot_width:350, plot_height:350}));
 
+.. _userguide_bokehjs_differences:
+
+Usage : Comparison to Python
+----------------------------
+
+While the BokehJS models generally match the Python equivalents, there are some cases
+where usage can differ significantly between BokehJS and the Python equivalent. Some
+instances we have identified are listed below.
+
+.. _userguide_bokehjs_differences_titles:
+
+Updating figure titles
+~~~~~~~~~~~~~~~~~~~~~~
+
+Calling ``var plot = new Bokeh.Plotting.figure({title:"some title"})`` replaces plot.title (a ``Bokeh.Title()``
+object by default) with the provided string. This makes it so the title cannot be dynamically updated later.
+In Python, the equivalent approach casts the string to the underlying Title() object.
+
+To preserve the ability to update the title later, use the following sequence instead:
+
+``var plot = new Bokeh.Plotting.figure()``
+``plot.title.text = "some title"``
+
+.. _userguide_bokehjs_differences_gridplots:
+
+Using gridplot layouts
+~~~~~~~~~~~~~~~~~~~~~~
+
+``Bokeh.Plotting.gridplot()`` does not inherit ``sizing_mode`` from the underlying plots. To match the expected
+behavior from the Python equivalent, for example when using ``sizing_mode: 'stretch_width'``, the ``sizing_mode``
+must be explicitly stated for both the ``figure()`` and the ``gridplot()`` objects.
+
+.. _userguide_bokehjs_differences_range_padding:
+
+Adjusting ``range_padding``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Setting ``plot.x_range.start`` and ``plot.x_range.end``, or ``plot.x_range.range_padding`` to adjust whitespace around
+glyphs on X axis is not persistent across resets. To match expected behavior from Python output, where the preferences
+will hold across figure resets, set both ``plot.x_range.range_padding`` and ``plot.x_range._initial_range_padding``.
+(Same for Y axis)
+
+.. _userguide_bokehjs_differences_palette:
+
+Palette for ``Bokeh.Plotting`` objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The color palette is available under ``Bokeh.Charts`` but not under ``Bokeh.Plotting``; to use with ``Bokeh.Plotting`` objects such as ``Bokeh.Plotting.image()``, the palette argument must be provided as a color string array, rather than as a string.
```

Comment:
The semicolon could be a period:
```suggestion
The color palette is available under ``Bokeh.Charts`` but not under ``Bokeh.Plotting``. To use it with ``Bokeh.Plotting`` objects such as ``Bokeh.Plotting.image()``, the palette argument must be provided as a color string array, rather than as a string.
```